README.md

```diff
@@ -5,7 +5,7 @@
 ---
 Connect Python SDK allows an easy and fast integration with Connect fulfillment API. Thanks to it you can automate the fulfillment of orders generated by your products.
 
-In order to use this library, please ensure that you have read first the documentation available on Connect knowladge base article located here, this one will provide you a great information on the rest api that this library implements.
+In order to use this library, please ensure that you have read first the documentation available on Connect knowledge base article located here, this one will provide you a great information on the rest api that this library implements.
```

Comment:
please, add proper links for "here"

Reply:
Since I didn't write the original text, I guess that the most appropriate link for that would the public API documentation? https://confluence.int.zone/display/CONNECT/Connect+Public+API

Reply:
Not at all. Please review the original readme of php-sdk, is a copy and paste